skaffold.hcl

```diff
@@ -0,0 +1,29 @@
+description = "Easy and Repeatable Kubernetes Development"
+test = "skaffold version"
+binaries = ["skaffold"]
+
+darwin {
+  source = "https://github.com/GoogleContainerTools/skaffold/releases/download/v${version}/skaffold-darwin-amd64"
```

Comment:
Can you `s,amd64,${arch},`? It looks like they have arm64 binaries!

Reply:
Actually you should be able to get rid of the darwin/linux blocks altogether - `/skaffold-${os}-${arch}` should work fine for this project.

Reply:
And you can use the same variables in the `rename` block.